mountinfo/mountinfo_unix.go

```diff
@@ -26,6 +41,11 @@ func mounted(path string) (bool, error) {
 		}
 	}
 
+	path, err = normalizePath(path)
```

Comment:
A bit on the fence if we should do the magic as part of this, or instead provide "normalizepath" as a utility, so that the caller can decide wether or not paths should be resolved

Reply:
Good idea about providing `NormalizePath()` -- perhaps this is the way to go.

Will implement

Reply:
In some cases though we don't need a normalized path (for example, when this is not a bind mount, a couple of stat() calls is sufficient and there's no need to perform "path normalization". There will be another optimization (see https://github.com/moby/sys/issues/25) that, also, does not require path normalization.

So if we ask a user to call it (in case they are not sure if the path is "normalized"), and one of the above optimization is used, the call is useless, thereas if we call it in the middle of `mounted()` like this patch does, the call is not useless.

I dunno :dog:

Reply:
Hmm.. right; should it be implemented as a filter/functional parameter that can be passed somewhere? (thinking out loud)

Reply:
What I think now is
1. with the optimizations in #25 we will hit mountinfo parser rarely;
2. parsing mountinfo is much heavier (in terms of time/resources) than normalizing one path, so I guess if we add it there, the effect is negligible.

Let me rebase this one on top of #25

Reply:
Sorry I mean #29 not #25 of course

Reply:
Ideally, this functionality can be enabled by a mere binary flag (`NormalizeMePlease`), but we don't have `flags` argument and I don't want to add one.

Reply:
Decided to close this one and add the commit to #29 instead.